### DISCUSSION TITLE
#21. VATZ biweekly meeting at 2022-11-09 08:00:00 -06:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#297] [General]
### 1. Overall
<img width="1211" alt="image" src="https://user-images.githubusercontent.com/6308023/198814564-3b607d6b-43de-4f2a-a3c4-41cf3e9d7b80.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
19 |  8/22 | 36
20 | 8/22 | 36

---

- Finished
   - #278
   - #282 
   - #85 
   - #285 
   - #281 
   - #277 
   - #228
   - #275 
    
- Move to Sprint 21
   - #87 
   - #119 
   - #121 
   - #122 
   - #146 
   - #175 
   - #187 
   - #197 
   - #202 
   - #238 
   - #242 
   - #261 
   - #280 
     
---

**Comment by xellos00** [comment]
# 동요

- #261
- #180

**Comment by heejin-github** [comment]
# 희진
- #294 
- #295

**Reply by xellos00** [reply]
@xellos00 
- [ ] Check, README.md is up to date.

**Comment by skonhwang** [comment]
# 승곤
- https://github.com/dsrvlabs/vatz/issues/294#issuecomment-1305325583
- https://github.com/dsrvlabs/vatz/issues/295#issuecomment-1308484508

**Comment by kim201212** [comment]
# 형기

- no issue.

**Comment by rootwarp** [comment]
# 준교
- #292 
- #298

**Reply by xellos00** [reply]
@xellos00 
- version.go

**Reply by hqueue** [reply]
e.g.
https://github.com/ethereum/go-ethereum/blob/master/params/version.go
https://github.com/cosmos/cosmos-sdk/blob/main/version/version.go

**Comment by gnongs** [comment]
## 진홍
- #269
- #280

**Comment by meetrick** [comment]
# 황제
- https://github.com/dsrvlabs/sei-project/issues/40

**Reply by xellos00** [reply]
@xellos00 
- [ ] Single plugin on/off w/o VATZ down.

**Comment by xellos00** [comment]
@qkrybin 
- Check the following repo, and create issue if you face any trouble. 
  - [ ] VATZ  
  - [ ] [vatz-plugin-sysutil](https://github.com/dsrvlabs/vatz-plugin-sysutil)
